windows/src/engine/keymanx64/keymanx64.cpp

```diff
@@ -311,6 +329,9 @@ BOOL SendPlatformComms32(WPARAM wParam, LPARAM lParam)
   HWND hwndLocalController = FindWindow(szWindowClass_x86_Wnd, NULL);
   if(hwndLocalController == NULL)
   {
+    // TODO: Perhaps we should trigger a shutdown here, because
+    // keyman.exe was not found. Right now, there is some confusion
+    // in regards to responsibility.
```

Comment:
```suggestion
    // TODO: Perhaps we should trigger a shutdown here, because
    // keyman.exe was not found. Right now, there is some confusion
    // in regards to responsibility. See #4976  
```